src/Assets/AssetsStorage.cpp

```diff
@@ -0,0 +1,54 @@
+#include "Assets/AssetsStorage.h"
+#include "GameManager.h"
+
+namespace Sourcehold
+{
+    namespace Assets
+    {
+        using namespace std;
+
+        const map<AssetStorageKey, AssetStorageLabel> AssetStorage::AssetStorageMapping = {
```

Comment:
if you declare something const, you can mark it as constexpr most of the time.

Reply:
📚

Reply:
📚